app/scripts/modules/openstack/pipeline/stages/scaleDownCluster/scaleDownClusterExecutionDetails.controller.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.scaleDownCluster.openstack.executionDetails.controller', [
+    require('angular-ui-router'),
+    require('../../../../core/delivery/details/executionDetailsSection.service.js'),
+    require('../../../../core/delivery/details/executionDetailsSectionNav.directive.js'),
+])
+  .controller('openstackScaleDownClusterExecutionDetailsCtrl', function ($scope, $stateParams, executionDetailsSectionService) {
+
+    $scope.configSections = ['scaleDownClusterConfig', 'taskStatus'];
+
+    function initialize() {
+      executionDetailsSectionService.synchronizeSection($scope.configSections);
+      $scope.detailsSection = $stateParams.details;
+    }
+
+    initialize();
+
+    $scope.$on('$stateChangeSuccess', initialize, true);
+
+  });
```

Comment:
Can you update this to follow the execution details controller changes that went in yesterday? There's a race condition we're trying to avoid.

In short, this controller should look like:
```
'use strict';

import detailsSectionModule from '../../../../core/delivery/details/executionDetailsSection.service';

let angular = require('angular');

module.exports = angular.module('spinnaker.core.pipeline.stage.scaleDownCluster.openstack.executionDetails.controller', [
  require('angular-ui-router'),
  detailsSectionModule,
  require('../../../../core/delivery/details/executionDetailsSectionNav.directive.js'),
])
  .controller('OpenstackScaleDownClusterExecutionDetailsCtrl', function ($scope, $stateParams, executionDetailsSectionService) {

    $scope.configSections = ['scaleDownClusterConfig', 'taskStatus'];

    let initialized = () => {
      $scope.detailsSection = $stateParams.details;
    };

    let initialize = () => executionDetailsSectionService.synchronizeSection($scope.configSections, initialized);

    initialize();

    $scope.$on('$stateChangeSuccess', initialize);

  });
```